README.rst

```diff
@@ -0,0 +1,16 @@
+lmfit
+*****
+
+This is a package providing a nice interface to nonlinear least-
```

Comment:
Please note somewhere here that this is an EUPS package so as not to confuse people about what is really in the repo.